utils/docs-helpers.ts

```diff
@@ -35,11 +35,9 @@ export const getVersionConfigPath = (filepath: string) => {
 export const getGithubURL = (filepath: string) => {
   const current = getVersion(filepath);
   const root = getVersionRootPath(filepath);
+  const ghIssuePath = `${NEXT_PUBLIC_GITHUB_DOCS}/issues/new?assignees=&labels=documentation&template=documentation.md`;
```

Comment:
Nitpick. 

```suggestion
  const ghIssuePath = `${NEXT_PUBLIC_GITHUB_DOCS}/issues/new?labels=documentation&template=documentation.md`;
```

I think we can omit assignees part here.

Reply:
Done. Thanks!